astropy/table/tests/test_operations.py

```diff
@@ -961,6 +961,105 @@ def test_mixin_functionality(self, mixin_cols):
                     'vstack unavailable' in str(err.value))
 
 
+class TestCStack():
+
+    def _setup(self, t_cls=Table):
+        self.t1 = t_cls.read([' a   b',
+                              ' 0. foo',
+                              ' 1. bar'], format='ascii')
+
+        self.t2 = t_cls.read([' a    b   c',
+                              ' 2.  pez  4',
+                              ' 3.  sez  5'], format='ascii')
+
+        self.t3 = t_cls({'a': [[5., 6.], [4., 3.]],
+                         'b': [['foo', 'bar'], ['pez', 'sez']]},
+                        names=('a', 'b'))
+        self.t4 = t_cls(self.t1, copy=True, masked=t_cls is Table)
+        self.t5 = t_cls({'a': [[4., 2.], [1., 6.]],
```

Comment:
One of the inputs should have a masked column with a masked element.

Reply:
added